### PR TITLE
Automatically share newly created quests with FabricateIO

### PR DESCRIPTION
**Pull request by toddmedema**
In case we need to debug or make changes

Note that the quests are discoverable in drive search, but don't automatically appear in the Fabricate drive - instead, you have to search them, add them to your drive, and THEN you'll be able to open it. That was the cause of the "Permission denied" bug I saw earlier tonight - it's just as true with quests you share with another person, until they add it to their drive, they'll get the same error.

### CODE REVIEW

File: app/actions/quest.tsx [hunk 1]
```diff
@@ -77,19 +76,34 @@ export function newQuest(user: UserState, dispatch: any) {
...
+              method: 'POST',
+              params: {sendNotificationEmails: false},
+              body: {
+                role: 'writer',
+                type: 'domain',
+                domain: 'Fabricate.io',
+                allowFileDiscovery: true
+              }
+        });
+        request.then((json: any, raw: any) => {
```
Comment (smartin015) [inline app/actions/quest.tsx]: Couldn't you just do `window.gapi.client.request({...}).then(...)`?

File: app/actions/quest.tsx [hunk 2]
```diff
@@ -77,19 +76,34 @@ export function newQuest(user: UserState, dispatch: any) {
...
+        }, (json: any) => {
+          console.log('Error connecting quest file to Fabricate.IO', json);
```
Comment (smartin015) [inline app/actions/quest.tsx]: Any chance we could log this to firebase/similar?

File: app/actions/quest.tsx (hunk 1 above)
Comment (toddmedema) [inline app/actions/quest.tsx]: True!

File: app/actions/quest.tsx (hunk 2 above)
Comment (toddmedema) [inline app/actions/quest.tsx]: Good idea - added some more logging of errors + user things to Google Analytics

**Pull request merged**